azurerm/internal/services/netapp/resource_arm_netapp_volume.go

```diff
@@ -176,6 +189,7 @@ func resourceArmNetAppVolumeCreateUpdate(d *schema.ResourceData, meta interface{
 			CreationToken:  utils.String(volumePath),
 			ServiceLevel:   netapp.ServiceLevel(serviceLevel),
 			SubnetID:       utils.String(subnetId),
+			ProtocolTypes:  utils.ExpandStringSlice(protocolTypes),
```

Comment:
This seems to imply multiple types can be passed in here?

Reply:
Yes. the SDK expects an array.